packages/sdk/app-framework/src/core/manager.test.ts

```diff
@@ -78,7 +78,7 @@ describe('PluginManager', () => {
     });
     const Test = definePlugin(testMeta, [Hello]);
 
-    const manager = new PluginManager({ plugins: [Test], enabled: [Test.meta.id], pluginLoader });
+    const manager = new PluginManager({ plugins: [Test], core: [], enabled: [Test.meta.id], pluginLoader });
```

Comment:
I think this one is not necessary, the only difference between core and enabled is that core cannot be disabled however this test doesn't attempt to disable the plugin